src/test/java/com/hotels/hcommon/ssh/session/DelegatingSessionFactoryBuilderTest.java

```diff
@@ -58,7 +60,14 @@ public void init() {
 
   @Test
   public void verifyNumberOfMethods() throws Exception {
-    assertThat(SessionFactoryBuilder.class.getDeclaredMethods().length, is(6));
+    Method[] declaredMethods = SessionFactoryBuilder.class.getDeclaredMethods();
```

Comment:
This change is needed since Jacoco seems to alter certain classes and add private methods to them which `getDeclaredMethods` was picking up and causing the test to fail, but only if `jacoco:report` was used via Maven, not in Eclipse. So instead I've gone for just checking the number of public methods which I think was the intention here anyway.